src/Log/Migrations/CreateNewLogTables.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Give\Log\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+
+
+/**
+ * Class CreateNewLogTables
+ * @package Give\Log\Migrations
+ *
+ * @since 2.9.7
+ */
+class CreateNewLogTables extends Migration {
+
+	/**
+	 * @return string
+	 */
+	public static function id() {
+		return 'create_new_log_table';
+	}
+
+	/**
+	 * @return int
+	 */
+	public static function timestamp() {
+		return strtotime( '2020-01-28 12:00' );
+	}
+
+
+	public function run() {
+		global $wpdb;
+
+		$sql = "CREATE TABLE {$wpdb->prefix}give_log (
+  			id BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  			log_type VARCHAR(16) NOT NULL,
+  			migration_id VARCHAR(64) NULL,
+  			data text NOT NULL,
+  			category VARCHAR(64) NULL,
+  			source VARCHAR(64) NULL,
+  			date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  			PRIMARY KEY  (id),
+            KEY log_type (log_type),
+            KEY migration_id (migration_id),
+            KEY category (category),
+            KEY source (source)
+		) {$wpdb->get_charset_collate()}";
```

Comment:
Please clean up the formatting.

Reply:
Hm... that's strange. In my editor it looks ok. I think this has something with Husky.

Reply:
Looks like it's a combination of tabs and spaces.

Reply:
Yeah! Resolved 0f4fbf72